packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -481,6 +489,24 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry, R
     emit AttestationExpirySecondsSet(_attestationExpirySeconds);
   }
 
+  /**
+   * @notice Setter for the metadata of an account.
+   * @param metadataURL The URL to access the metadata.
+   */
+  function setMetadataURL(string memory metadataURL) public {
```

Comment:
Doesn't look like this is being used anywhere else in the contract. Is this a requirement for every account and so should be set when creating one? If not, function should be `external`.

Reply:
Not every account, but external makes sense!